java/vector/src/main/java/org/apache/arrow/vector/BitVectorHelper.java

```diff
@@ -58,12 +58,27 @@ public static int bitIndex(int absoluteBitIndex) {
   public static void setValidityBitToOne(ArrowBuf validityBuffer, int index) {
     final int byteIndex = byteIndex(index);
     final int bitIndex = bitIndex(index);
-    byte currentByte = validityBuffer.getByte(byteIndex);
-    final byte bitMask = (byte) (1L << bitIndex);
+    int currentByte = validityBuffer.getByte(byteIndex);
+    final int bitMask = 1 << bitIndex;
     currentByte |= bitMask;
     validityBuffer.setByte(byteIndex, currentByte);
   }
 
+  /**
+   * Set the bit at provided index to 0.
+   *
+   * @param validityBuffer validity buffer of the vector
+   * @param index index to be set
+   */
+  public static void setValidityBitToZero(ArrowBuf validityBuffer, int index) {
```

Comment:
I know this mimics the existing setValidityBitToOne, but I think unsetBit would be a better name (we can consider adding an alias for setValidityBitToOne as "setBit" as well?

Reply:
Sounds good to me. Will provide the alias later.

Reply:
Alias provided. Please take a look. Thank you.